packages/edit-site/src/components/editor/use-adapt-editor-to-canvas.js

```diff
@@ -17,6 +22,8 @@ export function useAdaptEditorToCanvas( canvas ) {
 	} = useDispatch( editorStore );
 	const { get: getPreference } = useSelect( preferencesStore );
 	const registry = useRegistry();
+	const { resetZoomLevel } = unlock( useDispatch( blockEditorStore ) );
```

Comment:
The `useDispatch( blockEditorStore )` is also defined above. Let's move and combine them in single call.